### PR TITLE
Update alignment on Primary & Bannered Campaign Pages

### LINKED ISSUE #3022: Update Alignment on Primary & Bannered Campaign Pages
Update the alignment on the Bannered Campaign Pages so the user's eye doesn't have to bounce around so much.

- [x] Design has been finalized
- [x] Link to original design ticket: #2935
- [x] Link to Redpen: https://redpen.io/p/eu7e43fa328483dfb7
- [x] Design changes been merged into the master Sketch file (you can find this file in the Foundation Site file on the Campaigns page, designs are titled 'New Bannered Campaign Alignment')
- [x] Desktop and ~mobile~ medium mockups are included below (mobile should remain the same):

## Requirements Checklist:
- [x] "Cut out" should take up 8 columns
- [ ] Increase spacing between explainer text and subnav to 48px

Screen Shot 2019-04-17 at 11 11 52 AM

## Desktop
meet-our-fellows copy@2x

## Medium
Med  Full Width

### PR DESCRIPTION
**Pull request by mmmavis**
Closes #3022 and #3045

### Primary page

- full width: https://foundation-mofostaging-pr-3025.herokuapp.com/primary-page/
- full width (with secondary nav): https://foundation-mofostaging-pr-3025.herokuapp.com/en/about/
- narrow width: https://foundation-mofostaging-pr-3025.herokuapp.com/en/narrow-primary-page/
- narrow width (with secondary nav): https://foundation-mofostaging-pr-3025.herokuapp.com/en/narrow-primary-page-with-nav/

### Bannered Campaign page

- full width: https://foundation-mofostaging-pr-3025.herokuapp.com/en/campaigns/full-width-bannered-campaign/
- full width (with secondary nav): https://foundation-mofostaging-pr-3025.herokuapp.com/en/campaigns/full-width-bannered-campaign-with-nav/
- narrow width: https://foundation-mofostaging-pr-3025.herokuapp.com/en/campaigns/narrow-bannered-campaign/
- narrow width (with secondary nav): https://foundation-mofostaging-pr-3025.herokuapp.com/en/campaigns/narrow-bannered-campaign-with-nav/

### CODE REVIEW

**Comment by beccaklam** [comment]
Can we also increase the 15px padding on the right side to 24px?
Screen Shot 2019-04-18 at 2 16 26 PM

**Comment by mmmavis** [comment]
@kristinashu ^

**Comment by mmmavis** [comment]
@kristinashu ^ regarding the grey line

**Comment by kristinashu** [comment]
So I think we should keep the 15px on the right to allow for longer line length, I think it's rare that a letter will go right to the right edge.

Yes! Go ahead and remove that mysterious grey line!

**Comment by mmmavis** [comment]
@beccaklam if it works for you - I'm gonna file a follow-up for removing that grey line as it's a more complicated fix and will affect implementation of other page types as well.

**Comment by beccaklam** [comment]
> @beccaklam if it works for you - I'm gonna file a follow-up for removing that grey line as it's a more complicated fix and will affect implementation of other page types as well.

That works for me!

**Comment by mmmavis** [comment]
Follow-up ticket filed: https://github.com/mozilla/foundation.mozilla.org/issues/3047

**Comment by beccaklam** [comment]
Hey @mmmavis so for this PR, is the spacing fixed with your PR 3049? There should be about 24px between the intro paragraph and the menu:
## PR:
Screen Shot 2019-04-23 at 10 21 02 AM

 ## Design:
Screen Shot 2019-04-23 at 10 49 11 AM

**Comment by mmmavis** [comment]
@beccaklam no this PR is not up-to-date with `master` (aka with changes from other merged PRs) yet. It turns out this work is more complicated than I expected 'cuz the code is heavily shared between all page types. To avoid confusion, I'm gonna remove PR reviewers first and assign you back once it's ready for review again.

**Comment by beccaklam** [comment]
@mmmavis ok! 👌

**Comment by mmmavis** [comment]
@beccaklam This is ready for design review! Phew! I've included fixes for #3045 as well... hopefully I have captured everything.

Once this passes design review I'll assign dev for code review.

File: network-api/networkapi/wagtailpages/templates/wagtailpages/bannered_campaign_page.html [hunk 1]
```diff
@@ -7,13 +7,15 @@
...
+{% block content_wrapper_class %}{% endblock %}
+
 {% block campaign_page_header %}{% endblock %}
 {% block campaign_page_menu %}{% endblock %}
 
 Note: Wrapper HTML elements are copied from primary_page_menu.html as we want menu on BanneredCampaignPage to look exactly the same as PrimaryPage's.
 
 {% block minisite_title %}
-  <div id="multipage-nav-wrapper" class="hidden-sm-down full-width mb-4">
+  <div id="multipage-horizontal-nav-wrapper" class="hidden-sm-down full-width">
```
Comment (mmmavis) [inline network-api/networkapi/wagtailpages/templates/wagtailpages/bannered_campaign_page.html]: renamed this id name from `multipage-nav-wrapper` to `multipage-horizontal-nav-wrapper`

**Comment by mmmavis** [comment]
@youriwims this is ready for code review! 👀

**Comment by mmmavis** [comment]
Thanks @beccaklam @youriwims 🎉 I know this is a big PR to review!

**Pull request merged**